enterprise/static/enterprise/grant_data_sharing_permissions.js

```diff
@@ -1,16 +1,77 @@
+function togglePolicy (force_open) {
+    var policy = $(".consent-policy").first();
+    var icon = $("#consent-policy-dropdown-icon");
+    if ((!policy.is(":visible")) || force_open) {
+        policy.show();
+        icon.removeClass("fa-chevron-right").addClass("fa-chevron-down");
+        location.href = "#";
```

Comment:
I was made aware of a bug in some Webkit-based browsers where a single location.href call on its own does not take effect, but does if an empty fragment is called first.